react/training-project/src/app/screens/Profile/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import PrimaryLayout from '~components/PrimaryLayout';
+
+import ProfileDumb from './layout';
+
+function Profile({ userdata }) {
+  return (
+    <PrimaryLayout>
```

Comment:
Call just the Profile (layout), and inside of it wrap its content with the PrimaryLayout.